egs/librispeech/ASR/finetune_hubert_transducer/asr_datamodule.py

```diff
@@ -0,0 +1 @@
+/ceph-data4/yangxiaoyu/softwares/icefall_development/icefall_forked/egs/librispeech/ASR/tdnn_lstm_ctc/asr_datamodule.py
```

Comment:
Could you use a relative path?
You can refer to what we are doing for other folders with symlinks.